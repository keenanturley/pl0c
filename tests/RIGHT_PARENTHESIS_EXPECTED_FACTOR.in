var x, w;
begin
   x:= 4;
   w:= 7;
   if w > (x + 7 then
	w:= w + 1
   else
	w:= x;
end
