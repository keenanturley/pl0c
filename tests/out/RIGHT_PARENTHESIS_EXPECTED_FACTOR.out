Lexeme List:
29 2 x 17 2 w 18 21 2 x 20 3 4 18 2 w 20 3 7 18 23 2 w 13 15 2 x 4 3 7 24 2 w 20 2 w 4 3 1 18 2 w 20 2 x 18 22 19 

Lexeme List (Symbolic Representation):
varsym identsym x commasym identsym w semicolonsym beginsym identsym x becomessym numbersym 4 semicolonsym identsym w becomessym numbersym 7 semicolonsym ifsym identsym w gtrsym lparentsym identsym x plussym numbersym 7 thensym identsym w becomessym identsym w plussym numbersym 1 semicolonsym identsym w becomessym identsym x semicolonsym endsym periodsym 

Error: Right parenthesis expected to close currently opened expression factor.
